cluster-api/Gopkg.toml

```diff
@@ -38,27 +38,25 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  branch = "release-1.8"
+  branch = "release-1.9"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  branch = "release-1.8"
+  branch = "release-1.9"
 
 [[constraint]]
   name = "k8s.io/apiserver"
-  branch = "release-1.8"
+  branch = "release-1.9"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  branch = "release-5.0"
-  # revision = "1fb83015bea362f028381bc0fcfb2a4e3eb7390b"
+  branch = "release-6.0"
 
 # client-go needs this version
 [[constraint]]
   name = "github.com/Azure/go-autorest"
-  version = "v8.0.0"
+  version = "v9.1.0"
 
 [[constraint]]
   name = "github.com/kubernetes-incubator/apiserver-builder"
```

Comment:
What release do we use? I wouldn't want this to go in unless the apiserver builder has the fixes we need released. Then pin this to that release tag.

Reply:
I wanted to keep updating to 1.9 and the fix separate. 
Changed the git hash to version.